upcloud_api/cloud_manager/server_mixin.py

```diff
@@ -111,6 +111,7 @@ def create_server(self, server: Server) -> Server:
           * Debian 10: 01000000-0000-4000-8000-000020050100
           * Ubuntu 20.04: 01000000-0000-4000-8000-000030200200
           * Windows 2019: 01000000-0000-4000-8000-000010070300
+
```

Comment:
This was because `ruff` was complaining, and I too like a little bit of space so why not